packages/gatsby-source-wordpress/src/fetch.js

```diff
@@ -193,8 +193,7 @@ async function fetchData({
   _accessToken,
   _concurrentRequests,
 }) {
-  const type = route.type
-  const url = route.url
+  const { type, url } = route;
```

Comment:
Please remove the semicolon

Reply:
Semicolon has been removed resubmitted for review